ingest/pipeline/definition.json

```diff
@@ -57,5 +62,20 @@
       }
     ]
   }
+},
+{
+  "id": "apm_ingest_timestamp",
+  "body": {
+    "description": "Add an ingest timestamp for APM events",
+    "processors": [
+      {
+        "set": {
+          "if": "ctx.processor?.event != 'span'",
```

Comment:
How about also excluding `metric` and `profile` events until needed?

Reply:
I had considered that but figured they were low enough volume that it's ok?  I think building data frames on metric and profile data might actually be very useful and didn't want to raise a barrier to it.  Finally, since it's pretty straightforward to change this on the fly I wasn't too worried.  How strongly do you feel about the suggestion?

Reply:
Not strong at all, was mainly curious about why only excluding spans.